bigchaindb/tendermint/commands.py

```diff
@@ -44,7 +44,7 @@ def start():
     # start message
     logger.info(BANNER.format(bigchaindb.config['server']['bind']))
 
-    if int(getenv('BIGCHAINDB_START_TENDERMINT', 1)):
+    if int(getenv('BIGCHAINDB_START_TENDERMINT', 0)):
```

Comment:
`BIGCHAINDB_START_TENDERMINT` could just have a default value of `False` or  `0`, rather than doing a check like this here (and anywhere else it gets used). That way Tendermint won't get started unless they explicitly set it to `True` or `1` (either using an environment variable or a BigchainDB config file).

`BIGCHAINDB_START_TENDERMINT` should also be documented in the docs page about BigchainDB configuration settings, i.e. in `bigchaindb/docs/server/source/server-reference/configuration.md`

What will happen if `BIGCHAINDB_START_TENDERMINT` is truthy _and_ Tendermint is already running? Maybe there should be a check if Tendermint is already running, before running it again?

Reply:
> `BIGCHAINDB_START_TENDERMINT` could just have a default value of `False` or `0`, rather than doing a check like this here (and anywhere else it gets used). That way Tendermint won't get started unless they explicitly set it to `True` or `1` (either using an environment variable or a BigchainDB config file).

Yes, it should be in the config file, but →

> `BIGCHAINDB_START_TENDERMINT` should also be documented in the docs page about BigchainDB configuration settings, i.e. in `bigchaindb/docs/server/source/server-reference/configuration.md`

→ `BIGCHAINDB_START_TENDERMINT` was supposed to be a quick workaround, and →

> What will happen if `BIGCHAINDB_START_TENDERMINT` is truthy and Tendermint is already running? Maybe there should be a check if Tendermint is already running, before running it again?

→ this problem was not taken into account. To me, we can even remove the whole code that is starting tendermint.

Reply:
> To me, we can even remove the whole code that is starting tendermint.

Okay!

Reply:
@ttmc I think it will eventually get removed (soon) because production setup will start tendermint in seperate container and dev (by dev I mean users building ontop of BDB) will use some container management utility to setup the system so tendermint will have its own container there too.

Reply:
If you don't mind I can change the scope of this PR to remove that part of the code.